generators/generator-bot-enterprise-assistant/generators/app/templates/dialogs/BotTourDialog/language-generation/en-us/BotTourDialog.en-us.lg

```diff
@@ -169,13 +169,7 @@
          ]
       }
    ],
-   "selectAction":{
```

Comment:
Is this right? These got nothing to do with your tests right?

Reply:
Correct. I switched the PR to merge against `4.14` after forgetting that I had built against `main`. I'll re-submit the PR after cherry-picking the commits over.